.gitignore

```diff
@@ -25,6 +26,7 @@ build/
 bld/
 [Bb]in/
 [Oo]bj/
+[Ss]rc/[Tt]mp
```

Comment:
Why is this tmp folder needed?

Reply:
The packages are left there by the publish automation. @karajas can you make the publish job clean this up itself?

Reply:
I believe you can pass the intermediate path to the task to avoid this. https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs#L31 I suggest doing that.

Reply:
What Intermediate Path do you mean? Would that be where the packages get generated in bin/Product/.nuget, or would it be where I want the publish job to leave the packages that it publishes? If it's the second, wouldn't the same problem still be present?

Reply:
IndexDirectory is essentially the temp directory where index files get written so you can pass it a path under intermediate root to keep those files isolated out of the source tree.

Reply:
I don't you passing in the IndexDirectory yet.

Reply:
This ignore shouldn't be needed any longer.